.travis.yml

```diff
@@ -1,3 +1,22 @@
 language: nix
-script:
-  - bash ci/deploy.sh
+
+notifications:
+  irc:
+    on_success: change
```

Comment:
Would this duplicate our current notifications?

Reply:
Maybe we only need to make it notify on PRs or on failures.

Reply:
This will only write a notification on success, it the state wasn't success before

Reply:
Sounds good!

Reply:
Uh about the notification behavior on pull-requests I don't actually know, we have to figure that out, and disable it or modify it on the way.

Reply:
Not so important. I am usually pretty quick merging PRs